submissions/modupeAdeOnojobi/database/transportationQuery.sql

```diff
@@ -0,0 +1,34 @@
+1. SELECT survived, COUNT(*) FROM accidents WHERE survived = 1 GROUP BY survived;
+// 290
+
+
+2. SELECT survived, COUNT(*) FROM accidents WHERE survived = 0 GROUP BY survived;
+// 424
+
+
+3. SELECT P.full_name, P.sex, P.age, T.point_of_embarkation FROM passengers AS P LEFT JOIN trips AS T 
+	ON P.id = T.passenger_id WHERE P.age < 27 
+	AND (T.point_of_embarkation= 'Q' OR T.point_of_embarkation = 'C');
+
+
+
+4. SELECT COUNT(*), accidents.survived, trips.point_of_embarkation FROM accidents LEFT JOIN trips
```

Comment:
You are only asked to `count` here. This other columns `accidents.survived, trips.point_of_embarkation`, you weren't ask too